conanfile.py

```diff
@@ -12,7 +12,7 @@ class FreetypeConan(ConanFile):
     description = "FreeType is a freely available software library to render fonts."
     url = "http://github.com/bincrafters/conan-freetype"
     homepage = "https://www.freetype.org"
-    license = "BSD"
+    license = "FTL"
```

Comment:
FreeType is FTL AND GPL-2.0-only, so we should add both, I guess?

https://www.freetype.org/license.html

Reply:
sure